backend/src/main/java/gov/cdc/usds/simplereport/validators/TestResultFileValidator.java

```diff
@@ -0,0 +1,463 @@
+      ValueOrError testResult = getValue(row, "test_result", true);
+      ValueOrError orderTestDate = getValue(row, "order_test_date", true);
+      ValueOrError specimenCollectionDate = getValue(row, "specimen_collection_date", true);
+      ValueOrError testingLabSpecimenReceivedDate =
+          getValue(row, "testing_lab_specimen_received_date", true);
+      ValueOrError testResultDate = getValue(row, "test_result_date", true);
+      ValueOrError dateResultReleased = getValue(row, "date_result_released", true);
+      ValueOrError specimenType = getValue(row, "specimen_type", true);
+      ValueOrError orderingProviderId = getValue(row, "ordering_provider_id", true);
+      ValueOrError orderingProviderLastName = getValue(row, "ordering_provider_last_name", true);
+      ValueOrError orderingProviderFirstName = getValue(row, "ordering_provider_first_name", true);
+      ValueOrError orderingProviderMiddleName =
+          getValue(row, "ordering_provider_middle_name", false);
+      ValueOrError orderingProviderStreet = getValue(row, "ordering_provider_street", true);
+      ValueOrError orderingProviderStreet2 = getValue(row, "ordering_provider_street2", false);
+      ValueOrError orderingProviderCity = getValue(row, "ordering_provider_city", true);
+      ValueOrError orderingProviderState = getValue(row, "ordering_provider_state", true);
+      ValueOrError orderingProviderZipCode = getValue(row, "ordering_provider_zip_code", true);
+      ValueOrError orderingProviderPhoneNumber =
+          getValue(row, "ordering_provider_phone_number", true);
+      ValueOrError testingLabClia = getValue(row, "testing_lab_clia", true);
+      ValueOrError testingLabName = getValue(row, "testing_lab_name", true);
+      ValueOrError testingLabStreet = getValue(row, "testing_lab_street", true);
+      ValueOrError testingLabStreet2 = getValue(row, "testing_lab_street2", false);
+      ValueOrError testingLabCity = getValue(row, "testing_lab_city", true);
+      ValueOrError testingLabState = getValue(row, "testing_lab_state", true);
+      ValueOrError testingLabZipCode = getValue(row, "testing_lab_zip_code", true);
+      ValueOrError testingLabPhoneNumber = getValue(row, "testing_lab_phone_number", false);
+      ValueOrError pregnant = getValue(row, "pregnant", false);
+      ValueOrError employedInHealthcare = getValue(row, "employed_in_healthcare", false);
+      ValueOrError symptomaticForDisease = getValue(row, "symptomatic_for_disease", false);
+      ValueOrError illnessOnsetDate = getValue(row, "illness_onset_date", false);
+      ValueOrError residentCongregateSetting = getValue(row, "resident_congregate_setting", false);
+      ValueOrError residenceType = getValue(row, "residence_type", false);
+      ValueOrError hospitalized = getValue(row, "hospitalized", false);
+      ValueOrError icu = getValue(row, "icu", false);
+      ValueOrError orderingFacilityName = getValue(row, "ordering_facility_name", true);
```

Comment:
seems like we need to update the doc if we're going to require this regardless of `testing_lab_name`. same with all other `ordering_facility_*` fields
![Screen Shot 2022-10-03 at 3 36 53 PM](https://user-images.githubusercontent.com/6401323/193698310-c6afcd9f-e5a0-4ce8-87c1-f5e18e1c7616.png)

Reply:
Yeah, or update the docs to make one of them optional. But for now they both have the required tag in the docs